src/components/comments.md

```diff
@@ -3,323 +3,444 @@ title: Comments
-2. Comment form (or sign-in form if the user is not signed in)
-3. Comment sorting controls (described in [Recommended behaviour](#recommended-behaviour))
-4. The comments list
-5. An _"Add your comment"_ link pointing to the _"Add your comment"_ textarea in the form above the comments list
+The biggest challenge in terms of styling is the custom aesthetic for the sorting `<select>` element. The wtfforms.com site includes a solution using `appearance: none`[^8] and the positioning of a custom downwards-pointing arrow. The `<option>` elements cannot take many custom styles (only `color` and `background-color` are applicable) but this is considered a fair compromise given the robustness of the underlying native control.
 
-See the [Reference implementation](#reference-implementation) for an example layout implementation.
+```css
+.gel-comments__sort {
+  position: relative;
+}
 
-## Recommended behaviour
+.gel-comments__sort select {
+  -webkit-appearance: none;
+  appearance: none;
+  border: 0;
+  background: none;
+  padding-right: 1em;
+}
 
-Comments, and the ability to write them, should be considered a progressive enhancement and the functionality provided via JavaScript. In the [Reference implementation](#reference-implementation), some plain ES5 JavaScript is used to handle comment submissions and (re)render the comment stream. In practice, you are more likely to use a library like React or Vue, and fetch comment data over XHR. 
+.gel-comments__sort svg {
+  width: 0.75em;
+  position: absolute;
+  right: 0;
+  top: 0.25em;
+}
+```
 
-Where there is latency, a loading spinner may need to be incorporated similar to that implemented in the [Load more component](../load-more). The [Reference implementation](#reference-implementation) uses dummy data and just exemplifies the expected layout and interaction behaviour. Rendering is instantaneous.
+### Reply styling
 
-Where JavaScript is not available, the comment form's submit button is disabled and dimmed. The `<noscript>` tag for the comments stream is visible.
+As set forth in [the GEL Comments guide](https://www.bbc.co.uk/gel/guidelines/comments), reply comments should be demarcated using a vertical line down the left-hand side of the comment. It is recommended this is achieved using a `border` and not an image or gradient. CSS `border`s are honoured and visible in Windows High Contrast Mode.
 
-```html
-<div class="gel-comments__stream">
-  <noscript>You cannot read or write comments with JavaScript disabled. Please enable JavaScript and refresh the
-    page.</noscript>
-</div>
+```css
+.gel-comment--reply {
+  padding-left: 1rem;
+  border-left: 2px solid $gel-color--blue;
+}
 ```
 
+### State indication
+
+The pressed (`aria-pressed="true"`) state of controls must be indicated not just through a change in color[^9]. In the case of the like and dislike voting controls, the [reference implementation](#reference-implementation) uses a blue `fill` for the button's SVG icon.
```

Comment:
Less confusing? "...state of controls must not be indicated only through a change..."